test/unit/chains.test.ts

```diff
@@ -4,6 +4,8 @@ import { ChainAddressesSchema } from '../../src/types.js';
 
 import { expect } from 'chai';
 
+const ABACUS_WORKS_DEPLOYER_NAME = "Abacus Works";
```

Comment:
Do you think we should put this in `src` and export it properly? I was on the fence about it (for neutrality) but it seems like we'll use it in a few places

Reply:
yeah I think it makes sense, and probably include an `isAbacusWorksChain` helper function to clean things up